src/app/modal/modal.controller.js

```diff
@@ -151,36 +140,102 @@ class CareerPortalModalController {
+                if ((education[i].schoolName || education[i].startDate || education[i].endDate) && education[i + 1]) {
+                    this.linkedInData.resume += lineBreak;
+                }
+            }
+            // Add line break
+            this.linkedInData.resume += hardBreak;
+        }
+
         // Work Experience Block
-        this.linkedInData.resume = this.$filter('i18n')('modal.workExperience') + lineBreak;
+        this.linkedInData.resume += this.$filter('i18n')('modal.workExperience') + lineBreak;
         // Positions
         if (userProfile.positions) {
-            if (userProfile.positions.values && userProfile.positions.values.length) {
-                this.linkedInData.resume += (userProfile.positions.values[0].company.name || '') + ' ';
-                // Start Date
-                if (userProfile.positions.values[0].startDate) {
-                    this.linkedInData.resume += months[userProfile.positions.values[0].startDate.month - 1] + ' ' + userProfile.positions.values[0].startDate.year + ' - ' || '';
-                }
-                // End Date or 'Present'
-                if (userProfile.positions.values[0].endDate) {
-                    this.linkedInData.resume += months[userProfile.positions.values[0].endDate.month - 1] + ' ' + userProfile.positions.values[0].endDate.year || '';
-                } else {
-                    if (userProfile.positions.values[0].isCurrent) {
-                        this.linkedInData.resume += this.$filter('i18n')('modal.present');
+            var positions = userProfile.positions.values;
+            // Iterate through each position
+            if (positions && positions.length) {
+                for (var ii = 0; ii < positions.length; ii++) {
+                    // Add Employee section header
+                    this.linkedInData.resume += (positions[ii].company.name || '') + ' ';
+                    // Start Date
+                    if (positions[ii].startDate) {
+                        this.linkedInData.resume += months[positions[ii].startDate.month - 1] + ' ' + positions[ii].startDate.year + ' - ' || '';
+                    }
+                    // End Date or 'Present'
+                    if (positions[ii].endDate) {
+                        this.linkedInData.resume += months[positions[ii].endDate.month - 1] + ' ' + positions[ii].endDate.year || '';
+                    } else {
+                        if (positions[ii].isCurrent) { // jshint ignore:line
+                            this.linkedInData.resume += this.$filter('i18n')('modal.present');
+                        }
+                    }
+                    this.linkedInData.resume += lineBreak;
+                    // Title
+                    this.linkedInData.resume += positions[ii].title + lineBreak || '';
+                    // Industry
+                    this.linkedInData.resume += positions[ii].company.industry ? positions[ii].company.industry + lineBreak : '';
```

Comment:
In the tests I've run, if there's a startDate object, there's always a month field, That field may resolve as `''`, though.